atm.js

```diff
@@ -1,21 +1,145 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//Buttons DOM Variables
+var depCheck = $("#depCheck");
+var witCheck = $("#witCheck");
+var depSave = $("#depSave");
+var witSave = $("#witSave");
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+/////////////
+//Checkings//
+////////////
 
+//Create Checkings Constructor
+function Checkings(){
```

Comment:
I really like how you used a constructor function! I think that you could easily condense and the savings into one. maybe calling it account 
```
function Account(){
}
```